pkg/networkpolicy/controller.go

```diff
@@ -3,14 +3,17 @@ package networkpolicy
 import (
 	"context"
 	"fmt"
+	"runtime"
+	"strings"
 	"time"
+	"unique"
 
 	nfqueue "github.com/florianl/go-nfqueue"
 	"github.com/mdlayher/netlink"
-
 	v1 "k8s.io/api/core/v1"
```

Comment:
I prefer it with the blank line there.

(But we should adopt a consistent style. Currently:
  - `pkg/networkpolicy/controller.go` has a blank line between the github and k8s imports
  - `pkg/networkpolicy/metrics.go` puts them together
  - `cmd/main.go` is a mess

)